usr/bin/revpi-btuart

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# referenced pi-bluetooth, bluetooth attach for Revolution Pi BT
+. /usr/bin/revpi-common
+
+HCIATTACH=/usr/bin/hciattach
+TTYDEVNAME="$1"
+
+revpi_dev_type
+DTYPE=$?
+
+if [ "0" = $DTYPE ]; then
+  OPTS=""
+  TYPEID="any"
+  SPEED=""
+  FLOW=""
+  BDADDR=""
+elif [ "1" = $DTYPE ]; then
+  OPTS=""
+  TYPEID="any"
+  SPEED=""
+  FLOW=""
+  BDADDR=""
+else
+  # on Non-uart BT equiped device, this shouldn't reach, as this service is
+  # triggered by udev rule in 90-revpi-bluetooth.rules, which should not matched
+  exit 0
+fi
+
+$HCIATTACH $OPTS $TTYDEVNAME $TYPEID $SPEED $FLOW $BDADDR
```

Comment:
With this command it is totally opaque what is actually called. 
Why don't you follow the style they use here?
https://github.com/RPi-Distro/pi-bluetooth/blob/master/usr/bin/btuart